samples/browser/src/main/java/org/mozilla/samples/browser/DefaultComponents.kt

```diff
@@ -117,6 +120,8 @@ open class DefaultComponents(private val applicationContext: Context) {
         )
     }
 
+    var onFindPageClicked: (() -> Unit)? = null
```

Comment:
Having this lambda as global variable definitely needs another thought before integrating in r-b. We can address that in a follow-up though.